mem-12-q.xml

```diff
@@ -12340,8 +12340,8 @@ Tätä sanaa ei voida käyttää maassa olevaan reikään (josta katso {QemjIq:n
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also"></column>
-      <column name="notes">This idiomatic expression literally means "to sit in a chair". It is based on the homophony between {ba':v} and {-ba':v:suff}. It is used only condenscendingly in response to a statement that the speaker thinks is too obvious to have needed stating.</column>
-      <column name="notes_de">Dieser idiomatische Ausdruck bedeutet wörtlich "auf einem Stuhl sitzen". Es basiert auf der Homophonie zwischen {ba':v} und {-ba':v:suff}. Es wird nur kondensierend als Antwort auf eine Aussage verwendet, die der Sprecher für zu offensichtlich hält, um sie angeben zu müssen. [AUTOTRANSLATED]</column>
+      <column name="notes">This idiomatic expression literally means "to sit in a chair". It is based on the homophony between {ba':v} and {-ba':v:suff}. It is used only condescendingly in response to a statement that the speaker thinks is too obvious to have needed stating.</column>
+      <column name="notes_de">Dieser idiomatische Ausdruck bedeutet wörtlich "auf einem Stuhl sitzen". Es basiert auf der Homophonie zwischen {ba':v} und {-ba':v:suff}. Es wird nur abwertend als Antwort auf eine Aussage verwendet, die der Sprecher für zu offensichtlich hält, um sie angeben zu müssen. [AUTOTRANSLATED]</column>
```

Comment:
If the German note is correct following this change, the `[AUTOTRANSLATED]` tag should be removed.

Reply:
Since I haven't received a response on this, I'm going to merge it as-is. The tag can be removed later.